core/src/main/java/tc/oc/pgm/shops/Shop.java

```diff
@@ -23,7 +23,7 @@ public class Shop implements FeatureDefinition {
 
   private final String id;
   private final String name;
-  private final ImmutableList<Category> categories;
+  private final List<Category> categories;
```

Comment:
Why no longer immutable?
It being a definition, i'd ensure this stays immutable

Reply:
Whoops, my mistake! Must not have reverted from when I started to adjust based on Electroid's suggestions to make it generic. Latest commit reverts this.